src/kibana/components/agg_types/controls/precision.html

```diff
@@ -16,4 +16,16 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<div class="vis-option-item">
+  <label>
+    <input type="checkbox"
+    name="autoPrecision"
+    value="{{autoPrecision}}"
```

Comment:
You shouldn't need `value` or `ng-checked` here. Angular should take care of that based on the value of `ng-model`